source/includes/webhooks/_job_events.md

```diff
@@ -192,6 +192,162 @@ The Job Updated event is triggered any time one or more of the following fields
 |-----------|------|
 | `hiring_team` | Field contains the Greenhouse users responsible for this job.  Each field contains the user's Greenhouse user id and the user's external employee id from the customer's system.  The employee id will be null if it has not been set in Greenhouse.
 
+## Job opened
+
+```json
+{
+  "action": "job_opened",
```

Comment:
open_job_approval

Reply:
I can't change these. They're already designated event types that occur in the Change Log.